config/locales/en.yml

```diff
@@ -53,7 +53,8 @@ en:
       # end carrierwave messages
 
       uploads:
-        document_type_unknown:  'We couldn’t upload your file because it’s not a known document type.'
+        content_type_mismatch: "We couldn't upload your file because the content doesn’t match the extension."
```

Comment:
For some reason I was getting errors saying there was a syntax error when I attempted to use single quotes here